inst/include/sampler.h

```diff
@@ -551,6 +548,21 @@ namespace smc {
         }
     }
     
+	template <class Space>
+	void sampler<Space>::Multinomial(unsigned n, unsigned k, arma::vec w, unsigned int * X) {		
```

Comment:
It's been a long day, so apologies if I'm missing something, but do we even need this helper function? (By which I mean can we not just call rmultinom directly when it's used which I think is just 393/403 for resampling). It's not a big deal either way but less code is so often more...